examples/Training/java/src/training/HowToUseTables.java

```diff
@@ -114,6 +132,23 @@ private void createTable()
             table.addData(newRow);
             OriginalFile file = table.getOriginalFile(); // if you need to interact with the table
             file = new OriginalFileI(file.getId(), false);
+
+            DataManagerFacility dm = gateway.getFacility(DataManagerFacility.class);
+
+            FileAnnotation annotation = new FileAnnotationI();
+            annotation.setFile(file);
+            annotation.setNs(omero.rtypes
+                    .rstring(omero.constants.namespaces.NSBULKANNOTATIONS.value));
+            annotation = (FileAnnotation) dm.saveAndReturnObject(ctx, annotation);
```

Comment:
Good question, adding the link without the annotation on the server? Hmmm, should try it out locally. But was trying to be on the safe side, given that this is an example